### ISSUE TITLE
There was a problem with the initial load of your profile. Please try again. / There was a problem saving that account: Error: certificate has expired

### ISSUE THREAD

**Issue by abate** [turtl/tracker#371]
I downloaded the app, installed locally, create an account, confirmed the account and the I get this problem ? did I do anything wrong ?

**Comment by abate** [comment]
There was a small glitch at the account creation : 

"There was a problem saving that account: Error: certificate has expired " 

I've removed the account and started over. I can still see this problem.

But when I look at https://apiv3.turtlapp.com the certificate is ok ... On the console I I've only `INFO` messages, but no errors. I'm a bit puzzled. How can I debug this ?

Maybe these two errors are related ?

**Comment by Sufrostico** [comment]
Exact same problem here

**Comment by orthecreedence** [comment]
I'm not seeing this when creating test accounts, and it looks like the certs for both v0.7 and v0.6 servers are valid. Can you both confirm the server you're using in the login screen? Should be `https://apiv3.turtlapp.com`. Thanks!

**Comment by abate** [comment]
yes. I confirm : https://apiv3.turtlapp.com .  I can remove the account and try again. How can I debug this ?
